api/src/services/keycloak-service.ts

```diff
@@ -22,9 +22,20 @@ type BCEIDBusinessAttributes = BCEIDBasicAttributes & {
   display_name: [string];
 };
 
-interface KeycloakGetUserResponse {
-  users: KeycloakUser[];
-  roles: Record<string, string>[];
+interface KeycloakIDIRUserRecord {
+  username: string;
+  firstName: string;
+  lastName: string;
+  email: string;
+  attributes: {
+    idir_user_guid: string[];
```

Comment:
It might be the case that it is only ever 1 item, though we can't be certain, but these are based on the keycloak teams swagger, and they unfortunately don't specify anything more precise than just an array of strings.